src/AST/IMember.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace CppSharp.AST
+{
+    public interface IMember<M> where M : Declaration, IMember<M>
+    {
+        String Name { get; }
+        bool canOverride(M m);
```

Comment:
This method does not follow naming conventions (Uppercase) and I'd prefer for the parameter to have a meaningful name, like `member` or `decl`.